source/Dockerfile

```diff
@@ -2,7 +2,7 @@
 FROM alpine:3 as download-image
 RUN apk add unzip
 WORKDIR /tmp/pharo-images
-ADD https://files.pharo.org/image/110/Pharo11-SNAPSHOT.build.689.sha.8986777.arch.64bit.zip ./PharoImage.zip
```

Comment:
Maybe we can use https://files.pharo.org/image/120/Pharo12.0-SNAPSHOT.build.1570.sha.3dafb8a476.arch.64bit.zip and declare it as 12.0.1? Because it's almost a year of updates on top